Simonyi/Non-Standard-Operations/MTCS/MTPtg/MTPtg-Configuration-for-MTRot-MTMount.rst

```diff
@@ -0,0 +1,194 @@
+.. |author| replace:: *David Sanmartim*
+.. If there are no contributors, write "none" between the asterisks. Do not remove the substitution.
+.. |contributors| replace:: *Paulina Venegas*
+
+
+.. _rancher: https://rancher.cp.lsst.org/
+
+
+
+.. _MTMTPtg-Configuration-for-MTRotator-and-MTMount:
+
+#############################################
+MTPtg Configuration for MTRotator and MTMount
+#############################################
+
+
+.. note:: Important Information before start.
+
+    #. Only proceed if you are authorized to change the configuration of the CSC pointing component ``MTPtg`` of the Simonyi Telescope.
+   
+    #. A change of the ``MTMount`` CSC version needs to be done in *ArgoCD by a commissioning scientist*.
+
+    #. Remember use VPN.
+
+..
+
+.. note:: Important Information before start.
+    
+    Kubernetes authorization
+
+    1. To access the Kubernetes Cluster, you must have the necessary credentials installed on your computer.
+  
+            *Credentials*: Open a ticket directed to summit IT to access the Kubernetes Cluster, named **yagan**, through the rancher_. 
+
+    2. Once it is done, you can download your unique credential and place the :file:`yagan.yaml` file inside the :file:`~/.kube` directory in your local machine.
+
+..
+
+.. _MTMTPtg-Configuration-for-MTRotator-and-MTMount-Procedure-Overview:
+Overview
+========
+
+When either the ``MTRotator`` or ``MTMount`` components become unavailable during Simonyi operations, and you still want to continue testing and tracking,
+it's necessary to change the configuration of the ``MTPtg`` CSC to avoid entering into ``FAULT`` mode commanding these unavailable components.
+
+
+.. _MTMTPtg-Configuration-for-MTRotator-and-MTMount-Procedure-Error-Diagnosis:
+Error Diagnosis
+===============
+
+Cases
+-----
+
+1. ``MTRotator`` is not available, but you still want to track **without** the Rotator using the rest of the components; or you want to **include** 
+the Rotator in the tracking again.
+
+2. ``MTMount`` is not available (not starting up, for example), but you still want to use *CCW* + *Rotator*  **without** moving or commanding the mount, 
+or you want to revert the change and **include** the mount.
+
+
+.. _MTMTPtg-Configuration-for-MTRotator-and-MTMount-Procedure-Procedure-Steps:
+Procedure Steps
+===============
+
+.. warning:: 
+    
+    Announce through the Slack channel *#summit-simonyi* that the component is not available, and you are about to change the configuration.
```

Comment:
Maybe this part should be the first step of the procedure, adding a note that it should not be skipped